meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html

```diff
@@ -48,22 +48,20 @@
             <div class="mainbar__left">
                 <div class="image">
                     <div class="image__image">
-                        <img src="{{ project.image |thumbnail_url:'heroimage' }}" alt="{% if project.image_alt_text %}{{ project.image_alt_text }}{% else %}{% translate 'Here you can find a decorative picture.' %}{% endif %}" />
```

Comment:
I can't add a comment above but on line 43 there's a `FIXME`statement concerning where `layout-grid__area--maincontent` should be and it seems like they have fixed the issue in their last update so that can now be moved under the  `herounit-image-with-aside`element